### ISSUE TITLE
continuum process must be integrated!

### ISSUE THREAD

**Issue by dhuppenkothen** [dhuppenkothen/ShiftyLines#2]
At the moment, the continuum is evaluated in the middle of each bin. This strictly isn't correct.
The continuum process used must be integrated in the same way that the lines are over each energy bin.

**Comment by eggplantbren** [comment]
I wonder how much computational cost this will incur.

In the end, it might not make much difference. Plus, we could always just say the AR(1) process is our prior for the integrated values, rather than our prior for the underlying non-integrated function. It's a bit of a cheat, but might be fine.

**Comment by dhuppenkothen** [comment]
I've just rewritten and rerun an example, and it doesn't seem to make much difference. The current continuum model we're assuming is a simple power law, so calculating the integral of that is not really that much more expensive than calculating the function at the mid-bin value. The main reason why this matter is the fact that the bins don't have equal widths: bins at higher energies are wider, so assuming a function and evaluating that at the mid-bin values translates into model counts that don't follow the function we defined. 

At the moment, we've commented out the OU process for unrelated reasons, but we might add it back in the future.

**Comment by eggplantbren** [comment]
Ahh, I was mixing up the continuum with the OU part, though a similar argument could have applied. Glad that you implemented it already. Presumably it wasn't too costly, in which case there's no reason not to do things the more proper way.

**Comment by dhuppenkothen** [comment]
Thinking about it, I think it might actually be reasonable for the OU process *not* to be integrated, since it's supposed to measure calibration uncertainties, which I think might not necessarily depend on the width of each bin. Maybe @vicgrinberg has thoughts?

PS: I'm currently visiting @vicgrinberg and we're hard at work on this project, as you might have noticed :)

**Comment by vicgrinberg** [comment]
My current gut feeling is simply that we should leave the OU process out - we did use it originally to account for the fact that there were trends in continuum that the model we used for the continuum at that point (a constant) did not account for. If we decide to put it in, we should discuss it more and think more about it anyway (i.e. the observer, aka I, should be made to understand what the process exactly does).

Back to integrated continuum: it will be crucial in the future (i.e. not this paper and not this iteration of ShiftyLines), when we try to include proper physical continuum models. So we need to include it, even if there are computational costs to it.
